package.json

```diff
@@ -4,7 +4,7 @@
   "description": "JavaScript library for the Aragon Package Manager",
   "main": "src/index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest test/test.js --forceExit"
```

Comment:
Usage of ``forceExit`` is not recommended because it can hide some wrong behaviors.